src/meshes/p4est_mesh.jl

```diff
@@ -319,9 +319,9 @@ mesh from an Abaqus mesh file (`.inp`). Each element of the conforming mesh pars
 from the `meshfile` is created as a [`p4est`](https://github.com/cburstedde/p4est)
 tree datatype.
 
-To create a curved unstructured mesh `P4estMesh` two strategies are available:
+To create a curved/higher-order unstructured mesh `P4estMesh` two strategies are available:
 
-- `p4est_mesh_from_hohqmesh_abaqus`: High-order, curved boundary information created by
+- `p4est_mesh_from_hohqmesh_abaqus`: High-order, polygonial boundary information created by
```

Comment:
I would prefer to say curved here. My main reason is that this is the word used in the HOHQMesh and other software (like NekMesh) documentation to discuss the high-order boundary information. I have never encountered anyone referring to it as polygonal. Also, "polygonal" here makes it sound like you are representing the boundary information with polygons, which is not what is happening.

Reply:
I see your point. I thought it might make sense to note the difference between the higher-order elements without a mapping and those with a mapping, where only the latter are truly curved, while the other are piece-wise linear in some way.